src/SplitText.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const SplitText = ({ children, className, separator, ...rest }) => {
+    const words = children.split(separator);
+
+    return words.map((word, index) => {
+        const withSeparator = index !== words.length - 1;
+
+        return (
+            <span
+                key={ `word-${index}` }
+                className={ classNames('react-split-text', className) }
+                { ...rest }>
+                { word }
+                { withSeparator && separator }
```

Comment:
I had some problems with only passing a normal string space.
Are we sure it is rendering properly?
I previously used: `'\u00A0'`

Reply:
I've tried it in the demo and it renders properly, passing a space (I've changed the child border to green): 

![split](https://user-images.githubusercontent.com/33324750/80197776-07b69f00-8617-11ea-806b-da2d2051ff66.png)

That's the expected behaviour.

I know you were using `\u00A0` and it was fine, but as we want to support different separators we can't hardcode that there, and there is no need.

Reply:
Actually you were right, it wasn't rendering properly as it was rendering two spaces (the second one doesn't have a width or height but you can see the little green dot after the first one. 

This has been fixed by https://github.com/moxystudio/react-split-text/pull/1#discussion_r414445039.